web/react/pages/channel.jsx

```diff
@@ -70,8 +69,8 @@ function setupChannelPage(props) {
     // Modals
     //
     ReactDOM.render(
-        <GetLinkModal />,
-        document.getElementById('get_link_modal')
+        <GetTeamInviteLinkModal />,
```

Comment:
Yeah there's a few modals that are still global that will need to go there like the ImportThemeModal and DeletePostModal as well